Sources/Foundation/Process.swift

```diff
@@ -1117,7 +1117,11 @@ open class Process: NSObject {
     // convenience; create and launch
     open class func launchedProcess(launchPath path: String, arguments: [String]) -> Process {
         let process = Process()
-        process.launchPath = path
+        if #available(OSX 10.13, *) {
```

Comment:
Why we are not using the modern macOS here? Is it because macOS is not recognized here?
```suggestion
        if #available(macOS 10.13, *) {
```